releasenotes/notes/cloudkitty-fix-es-to-os-migration-b0efd2626f59d977.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fixes the bulk API of CloudKitty so that it now supports the migration
+    from Elasticsearch to OpenSearch.
```

Comment:
Do we need a new cloudkitty image?

Reply:
Yes we will need to build for both Zed and Antelope.

Reply:
PRs here to bring in the fixes:
https://github.com/stackhpc/cloudkitty/pull/62
https://github.com/stackhpc/cloudkitty/pull/63